src/components/PersonRow.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { useParams } from 'react-router';
+
+import classNames from 'classnames';
+
+import PersonName from './PersonName';
```

Comment:
You don't need blank lines between these imports
```suggestion
import React from 'react';
import { useParams } from 'react-router';
import classNames from 'classnames';
import PersonName from './PersonName';
```